src/LessonPreview/LessonPreview.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import "./LessonPreview.scss";
+import QuestionCard from "./QuestionCard/QuestionCard";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { setLesson } from "../actions";
+
+const LessonPreview = ({
+  questions,
+  setQuestions,
+  lessonTitleText,
+  setLessonTitleText,
+  deleteQuestion,
+  setLesson,
+}) => {
+
+  const createLesson = (e) => {
+    e.preventDefault();
+    const lesson = {
+      name: lessonTitleText,
+      questions: questions,
+    };
+    setLesson(lesson);
```

Comment:
Final functionality that sends the lesson object to Redux and to the BE once functionality is completed